docs/backend/guides/service-objects/structure.md

```diff
@@ -0,0 +1,145 @@
+:::
+
+## Data/Utility Service Objects High Level Structure
+
+Data and utility service objects follow the same structure pattern, with just naming being different, so they'll be 
+covered together at first.
+
+In the `mymove` `./pkg/services` directory you should see several subdirectories (`Go` subpackages) and singular 
+`Go` files at the end. The names of these solo `Go` files should match that of a directory above.
+
+```text {7,11}
+mymove/
+├── pkg/
+│   ├── services/
+│   │   ├── ...
+│   │   ├── mto_agent/
+│   │   ├── mto_service_item/
+│   │   ├── mto_shipment/       <- a subpackage
+│   │   ├── ...
+│   │   ├── mto_agent.go
+│   │   ├── mto_service_item.go
+│   │   ├── mto_shipment.go     <- the interface in the services package
+│   │   ├── ...
+```
+
+### Data Service Objects High Level Structure
+
+The names of the file + subpackage should match the database table name, but in singular form. E.g. `mto_shipment` 
+service object for the `mto_shipments` table.
+
+Each of these subpackages handles the operations for one particular data model. This helps us keep track of our
+database interactions and allows for different parts of the codebase to interact with models while re-using the same 
+validation and functionality.
+
+### Utility Service Objects High Level Structure
+
+The names of the file + subpackage should match the function the service object serves, e.g. `fetch`.
+
+Each of these subpackages should focus on the functionality they're meant to serve. It helps us have logic we can 
+re-use across service objects.
+
+## Orchestrator Service Objects High Level Structure
+
+Orchestrator service objects are similar overall to the other types of service objects, but the top level structure 
+and naming are slightly different. 
+
+We have an `orchestrators` subpackage inside `services` that holds the subpackages for each orchestrator. This helps 
+us separate out the `orchestrators` from the other more "regular" service objects (i.e. pre-existing service object 
+types that existed before `orchestrators` were added). 
+
+As for the corresponding interface in the `services` package, we use a standalone file with the orchestrator name 
+followed by `_orchestrator` to identify it more easily as an orchestrator.
+
+```text {6,7,11}
+mymove/
+├── pkg/
+│   ├── services/
+│   │   ├── ...
+│   │   ├── office_user/
+│   │   ├── orchestrators/            <- all orchestrators subpackage
+│   │   │   ├── shipment              <- subpackage for a single orchestrator
+│   │   ├── order/
+│   │   ├── ...
+│   │   ├── reweight.go
+│   │   ├── shipment_orchestrator.go  <- the interface in the services package for a single orchestrator
+│   │   ├── sit_extension.go
+│   │   ├── ...
+```
+
+So in our example above, we have an orchestrator service object called `shipment` that lives in the `orchestrators` 
+subpackage. It has a corresponding interface file called `shipment_orchestrator` in the `services` package.
```

Comment:
**Suggestion**: It could be helpful to relate the orchestrator concept back to the Cat example from earlier

Reply:
I think this would be great! Do you happen to have an example in mind? The main thing I can think of right now would be if we tweaked the sample `Cat` model to be a child model of a more generic `Pet` model. That could maybe also set us up to have good examples of setting up an orchestrator as well.

Reply:
I ended up moving toward a `Pet`/`Cat` parent/child models and updated the docs. This should hopefully make it easier to show orchestrators and to note their differences.